api_app/schemas/workspace.json

```diff
@@ -22,6 +23,12 @@
           "description": "Description of the workspace",
           "updateable": true
       },
+      "app_id": {
```

Comment:
Not clear why this is being added. This means every workspace will need an app id passing into the API and the UI will require it. Not sure if this is the intention?